string[] names
bool useSensors
---
float32[] jointAngles